docs/contributing/01-start-here/05-development/01-development.md

```diff
@@ -60,7 +60,6 @@ pnpm turbo <task> --filter=<project> -- <args>
 [volta]: https://volta.sh
 [PNPM]: https://pnpm.io
 [Docker]: https://docs.docker.com/get-docker/
-[emscripten]: https://emscripten.org/docs/getting_started/downloads.html
```

Comment:
Why is this being removed?

Reply:
Ah yeah, if I saw it correctly it was referenced nowhere

Reply:
@Lancear once you resolve this one, we will get this change in!!!

Reply:
yeah should we keep that link if its not used nor visible?

Reply:
Good catch - let's remove it. It was added in https://github.com/winglang/wing/pull/1249/files but it is not used anymore. Resolving.